src/Sulu/Bundle/AdminBundle/Resources/js/components/BlockCollection/SortableBlock.js

```diff
@@ -39,14 +85,6 @@ class SortableBlock<T: string, U: {type: T}> extends React.Component<Props<T, U>
         }
     };
 
-    handleRemove = () => {
```

Comment:
do we need to keep this as methods are public in js? or can we safely remove it?

Reply:
it is unusual to call a method of a component (you would need to get the ref) and react recommends to use composition over inheritance, so i would say its safe to remove it 🙂